src/components/tx/DecodedTx/index.tsx

```diff
@@ -76,8 +58,8 @@ const DecodedTx = ({
   let toInfo = tx && {
     value: tx.data.to,
   }
-  if (txDetails && isCustomTxInfo(txDetails.txInfo)) {
-    toInfo = txDetails.txInfo.to
+  if (txDetails && isCustomTxInfo(txDetails?.txInfo)) {
+    toInfo = txDetails?.txInfo.to
   }
 
   const decodedDataBlock = <DecodedData txData={txData} toInfo={toInfo} />
```

Comment:
I am seeing all the data twice inside DecodedTx

<img width="661" alt="Screenshot 2024-09-17 at 15 49 57" src="https://github.com/user-attachments/assets/df510fcc-721e-4e7a-9ef1-136aba46f0a3">

Reply:
@usame-algan can you please check it again?

Reply:
Fixed 🙏